incubator/fluentd-cloudwatch/templates/_helpers.tpl

```diff
@@ -2,15 +2,16 @@
 {{/*
 Expand the name of the chart.
 */}}
-{{- define "fluentd-cloudwatch.name" -}}
-{{- default .Chart.Name .Values.nameOverride | trunc 24 -}}
+{{- define "name" -}}
```

Comment:
You'll want to keep the chart name prefix, more info @ https://github.com/kubernetes/charts/issues/1785